src/quizzes/chapter-3/3.1/immutable-infra-and-packer-quiz.js

```diff
@@ -0,0 +1,46 @@
+const rawQuizdown = `
+---
+shuffleAnswers: true
+shuffleQuestions: true
+---
+
+# Which statement is true about immutable infrastructure as opposed to mutable infrastructure?
+
+1. [x] Immutable infrastructure has less risk and is less complex
+1. [ ] Immutable infrastructure has more risk but is less complex
+	> It might prove beneficial to go watch the [What is Mutable vs. Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure/) video again
+1. [ ] Immutable infrastructure has less risk but is more complex
+	> It might prove beneficial to go watch the [What is Mutable vs. Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure/) video again
+1. [ ] Immutable infrastructure has more risk and is more complex
+	> It might prove beneficial to go watch the [What is Mutable vs. Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure/) video again
```

Comment:
While the term "less complex" is rather broad here, it was referenced directly in the video, and refers to removing the possibility of random "in between" versions that mutable infra poses due to certain upgrades working and others failing. With immutable, you either get version 1 or 2, not some weird in between one

Reply:
I appreciate this comment